cron/data/request.pb.go

```diff
@@ -0,0 +1,180 @@
+// Code generated by protoc-gen-go. DO NOT EDIT.
```

Comment:
Add a README.md file on how to generate this (command line used to generate this).

Reply:
Done.